src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -4767,6 +4804,263 @@ namespace OutputReportTabular {
+            } else if (TESCoil(iCoil).CondenserType == EvapCooled) {
+                SysTotalHVACRejectHeatLoss += TESCoil(iCoil).EvapCondPumpElecConsumption + TESCoil(iCoil).ElectEvapCondBasinHeaterEnergy +
+                                              TESCoil(iCoil).EvapWaterConsump * RhoWater * H2OHtOfVap_HVAC - TESCoil(iCoil).Q_Ambient;
+            }
+        }
+
+        // Water heater and thermal storage
+        for (iTank = 1; iTank <= NumWaterThermalTank; ++iTank) {
+            if (WaterThermalTank(iTank).AmbientTempIndicator == AmbientTempOutsideAir) {
+                SysTotalHVACRejectHeatLoss += WaterThermalTank(iTank).FuelEnergy - WaterThermalTank(iTank).TotalDemandEnergy;
+            }
+        }
+
+        // Variable Refrigerant Flow
+        for (iCoil = 1; iCoil <= NumVRFCond; ++iCoil) {
+            if (VRF(iCoil).CondenserType == AirCooled) {
+                SysTotalHVACRejectHeatLoss += VRF(iCoil).CoolElecConsumption + VRF(iCoil).HeatElecConsumption +
+                                              VRF(iCoil).CrankCaseHeaterElecConsumption + VRF(iCoil).DefrostConsumption +
+                                              (VRF(iCoil).TotalCoolingCapacity - VRF(iCoil).TotalHeatingCapacity) * TimeStepZoneSec;
+            } else if (VRF(iCoil).CondenserType == EvapCooled) {
+                SysTotalHVACRejectHeatLoss += VRF(iCoil).EvapCondPumpElecConsumption + VRF(iCoil).BasinHeaterConsumption +
+                                              VRF(iCoil).EvapWaterConsumpRate * TimeStepZoneSec * RhoWater * H2OHtOfVap_HVAC;
+            } else if (VRF(iCoil).CondenserType == WaterCooled) {
+                SysTotalHVACRejectHeatLoss += VRF(iCoil).QCondEnergy;
+            }
+        }
+
+        // Refrigerated Rack
+        for (iRef = 1; iRef <= NumRefrigeratedRacks; ++iRef) {
+            if (RefrigRack(iRef).CondenserType == AirCooled) {
+                SysTotalHVACRejectHeatLoss += RefrigRack(iRef).RackElecConsumption + RefrigRack(iRef).RackCoolingEnergy;
+            } else if (RefrigRack(iRef).CondenserType == EvapCooled) {
+                SysTotalHVACRejectHeatLoss += RefrigRack(iRef).EvapPumpConsumption + RefrigRack(iRef).BasinHeaterConsumption +
+                                              RefrigRack(iRef).EvapWaterConsumption * RhoWater * H2OHtOfVap_HVAC;
+            } else if (RefrigRack(iRef).CondenserType == WaterCooled) {
+                SysTotalHVACRejectHeatLoss += RefrigRack(iCoil).CondEnergy;
+            }
+        }
+
+        // Refrigerated Case - Condenser
+        for (iRef = 1; iRef <= NumRefrigCondensers; ++iRef) {
+            SysTotalHVACRejectHeatLoss += RefrigRack(iCoil).CondEnergy;
+        }
+
+        // Evaporative coolers
+        for (iCooler = 1; iCooler <= NumEvapCool; ++iCooler) {
+            SysTotalHVACRejectHeatLoss += EvapCond(iCooler).EvapWaterConsump * RhoWater * H2OHtOfVap_HVAC + EvapCond(iCooler).EvapCoolerEnergy;
```

Comment:
@XuanLuoLBL When all unit tests are run serially, this line fails with an array bounds error. There is no `clear_state` function for EvaporativeCoolers to reset NumEvapCool to zero and reset or deallocate other variables. If you need help adding this, let me know.